### DISCUSSION TITLE
Process matrix for dissipative propagation?

### DISCUSSION THREAD

**Discussion by will-atom** [JuliaQuantumControl/QuantumControl.jl#47] [Q&A]
How can I extract an effective process matrix for the evolution of a pair of qubit levels embedded in a larger Hilbert space, such as the transmon qubit example? A quick search of the documentation does not turn up the words "process matrix," though I'm sure this is straightforwardly possible.

**Comment by goerz** [comment]
I have not implemented anything like that, and it would probably be beyond the scope of the package. You will have to implement this yourself (or maybe some other package like QuantumOptics already has something to help with this.)
